specification/common-types/data-plane/v1/types.json

```diff
@@ -70,5 +77,40 @@
       },
       "description": "The resource management error additional info."
     }
+  },
+  "responses": {
+    "ErrorResponse": {
+      "description": "Error response for all Azure services to return error details for failed operations.",
+      "headers": {
+        "x-ms-error-code": {
+          "type": "string",
+          "description": "The error code"
+        }
+      },
+      "schema": {
+        "$ref": "#/definitions/ErrorResponse"
+      }
+    }
+  },
+  "parameters": {
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "The API version to use for this operation.",
```

Comment:
@mikekistler -- This should be more descriptive, e.g. the same format as our date based version, etc.